android-core/src/main/java/com/mparticle/modernization/BatchManager.kt

```diff
@@ -0,0 +1,13 @@
+package com.mparticle.modernization
+
+class BatchManager(private val api : MpApiClientImpl) {
```

Comment:
Same as other comment, this should take an `ApiClient` interface of which `MpApiClientImpl` extends right?